source/api.rst

```diff
@@ -205,8 +205,8 @@ Possible scenarios when this channel notifies are: the transaction now is confir
 
 **partialAdded/{address}**
 
-The partialAdded channel notifies when an AggregateBondedTransaction related to an address reaches the partial state, waiting to have all required cosigners.
-The messages returned contain information about the transactions.
+The ``partialAdded`` channel notifies subscribed clients when an :ref:`AggregateBondedTransaction <aggregate-bonded>` related to the given address enters the partial state, waiting to have all required cosignatures.
```

Comment:
perhaps:

waiting for required cosignatures to complete the :ref:`AggregateBondedTransaction <aggregate-bonded>`